packages/f/font-ubuntu-sans-ttf/package.yml

```diff
@@ -0,0 +1,15 @@
+name       : font-ubuntu-sans-ttf
+version    : 1.006
+release    : 1
+source     :
+    - https://github.com/canonical/Ubuntu-Sans-fonts/archive/refs/tags/v1.006.tar.gz : ddd9e2b74767cf2b5b33d933ff8d579e806b0b26e9b7abf96ebfe9b3f6895a65
+homepage   : https://github.com/canonical/Ubuntu-Sans-fonts
+license    : Ubuntu Font License-1.0
```

Comment:
The correct identifier is `Ubuntu-font-1.0` according to [spdx](https://spdx.org/licenses/)